quinn/append_if_schema_identical.py

```diff
@@ -0,0 +1,33 @@
+from pyspark.sql import DataFrame, SparkSession
+from pyspark.sql.types import StructType, StructField, IntegerType, StringType, DoubleType
+
+
+class SchemaMismatchError(ValueError):
+    """raise this when there's a schema mismatch between source & target schema"""
+
+
+def append_if_schema_identical(source_df: DataFrame, target_df: DataFrame):
```

Comment:
Please provide a docstring in [sphinx format](https://github.com/MrPowers/quinn#code-style). Opposite this functions worn't be shown in API documentation.